### ISSUE TITLE
[10.0][stock_card] 

### ISSUE THREAD

**Issue by alan196** [Vauxoo/addons-vauxoo#1249]
https://github.com/Vauxoo/addons-vauxoo/tree/10.0/stock_card
I install the module stock_card  and when I try to show the moves I get the following error:
```
Traceback (most recent call last):
  File "/home/odoo-10.0/odoo/odoo/http.py", line 640, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/home/odoo-10.0/odoo/odoo/http.py", line 677, in dispatch
    result = self._call_function(**self.params)
  File "/home/odoo-10.0/odoo/odoo/http.py", line 333, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/home/odoo-10.0/odoo/odoo/service/model.py", line 101, in wrapper
    return f(dbname, *args, **kwargs)
  File "/home/odoo-10.0/odoo/odoo/http.py", line 326, in checked_call
    result = self.endpoint(*a, **kw)
  File "/home/odoo-10.0/odoo/odoo/http.py", line 935, in __call__
    return self.method(*args, **kw)
  File "/home/odoo-10.0/odoo/odoo/http.py", line 506, in response_wrap
    response = f(*args, **kw)
  File "/home/odoo-10.0/odoo/addons/web/controllers/main.py", line 889, in call_button
    action = self._call_kw(model, method, args, {})
  File "/home/odoo-10.0/odoo/addons/web/controllers/main.py", line 877, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/home/odoo-10.0/odoo/odoo/api.py", line 689, in call_kw
    return call_kw_multi(method, model, args, kwargs)
  File "/home/odoo-10.0/odoo/odoo/api.py", line 680, in call_kw_multi
    result = method(recs, *args, **kwargs)
  File "/home/odoo-10.0/addons-vauxoo/stock_card/model/stock_card.py", line 49, in stock_card_move_get
    return self.action_view_moves()
  File "/home/odoo-10.0/addons-vauxoo/stock_card/model/stock_card.py", line 356, in action_view_moves
    self._cr, self._uid, 'stock_card', 'stock_card_move_action')
TypeError: unbound method get_object_reference() must be called with ir.model.data instance as first argument (got Cursor instance instead)
```
I see the commit history and this module is not migrated it only have `installable: True` in the manifest file.
@nhomar @moylop260 
Do you have plans to migrate this module? or any other way to obtain this information?
I need this in V10 and V11. I will migrate this module and propose PR.
But I sent this Issue in order to avoid work duplication.

**Comment by nhomar** [comment]
Hello.

This modules will be migrated only for V11.0 (not v10.0).

Why? on v 11.0 this report is partially available, what we do on v10.0 will be backported what we will have on v11.0 in all case.

Regards.

**Comment by alan196** [comment]
Thanks for your response @nhomar 
How can we help to migrate this module?
Do you have a roadmap?
